frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -280,7 +280,7 @@ local footerTextGeneratorMap = {
         local symbol_type = footer.settings.item_prefix or "icons"
         local prefix = symbol_prefix[symbol_type].book_time_to_read
```

Comment:
Probably simpler to just fix it here?

e.g., `local prefix = blah or ""`

Reply:
I did try that, and it works for the bug around line 280, but I was trying to keep things consistent with e.g. 

```
    letters = {
        time = nil,

time = function(footer)
...
        if not prefix then
            return clock
        else
            return prefix .. " " .. clock
        end
```

letters.time is set to null and has a wrapper.

Which would be better?

Reply:
Keep it as it is :) (@NiLuJe trick would have had a space added to the prefix, so we would no longer (yay!) get the empty string we want)

Reply:
> @NiLuJe trick would have had a space added to the prefix

Oh, quite right. Oops ;p.